OleksiiBozhchenko/HW6/Task1.py

```diff
@@ -0,0 +1,30 @@
+# In the range from 1 to 10 determine
+#  - even number that are divisible by 2,
+#  - odd numbers which are divisible by 3,
+#  - numbers that are not divisible by 2 and 3.
+
+# #
+# # Example 1
+# #
+even = []
+odd = []
+other = []
+
+for i in range(1, 10):
+    if i%2 == 0:
+        even.append(i)
+    elif i%3 == 0:
+        odd.append(i)
+    else:
+        other.append(i)    
+
+print(f"Even numbers are: {even}\nOdd numbers are: {odd}\nNon divisible numbers are: {other}")
+
+# #
+# # Example 2 
+# # (In this example I experimented with List Comprehension)
+# #
```

Comment:
you output:
Even numbers are: [2, 4, 6, 8]
Odd numbers are: [3,      ??? 6 ???  , 9]
Non divisible numbers are: [1, 5, 7]

another implementation:

l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')